src/helm/benchmark/static/benchmarking.js

```diff
@@ -479,6 +462,31 @@ $(function () {
     return instanceKeyToDiv;
   }
 
+  function loadAndRenderRequests(runSpec, instanceKeyToDiv, predictedIndex) {
+    if (runSpecsNamesWithLoadedRequests.includes(runSpec.name)) {
+      return;
+    }
+    runSpecsNamesWithLoadedRequests.push(runSpec.name);
+    $.getJSON(scenarioStateJsonUrl(suite, runSpec.name), {}, (scenarioState) => {
+      scenarioState.request_states.forEach((requestState) => {
+        // For the adapter method multiple_choice_separate_calibrated,
+        // only show the original requests and skip the rest
+        if (requestState.request_mode == "calibration") {
```

Comment:
Should we exclude this on the backend if we're moving more logic there?

Reply:
Good point. Deleted this.